src/VirtoCommerce.NotificationsModule.Data/Migrations/20190528144557_InitialNotifications.cs

```diff
@@ -266,15 +276,33 @@ INNER JOIN
 			                            END
 	                            END
                             FROM [PlatformNotificationTemplate] pnt
-                            INNER JOIN 
+                            INNER JOIN pnt2

```

Comment:
We could lost data. e.g. Look at screen
![image](https://user-images.githubusercontent.com/8361000/97416787-916d5f00-190f-11eb-9dd9-42980857c7f2.png)

Reply:
I'll look at this.

Reply:
I didn't touch this. It was made before. Looks it like made specially:
``` sql
    CASE WHEN pnt.[NotificationTypeId] LIKE '%EmailNotification%' THEN [Body] ELSE '' END,
	CASE WHEN pnt.[NotificationTypeId] LIKE '%SmsNotification%' THEN [Body] ELSE '' END,
```

Reply:
agree it's an old bug, maybe need to find solution